sample/src/main/java/com/nerdstone/neatform/form/FormRecyclerAdapter.kt

```diff
@@ -27,7 +28,7 @@ class FormRecyclerAdapter : RecyclerView.Adapter<FormRecyclerAdapter.MainViewHol
     override fun onBindViewHolder(holder: MainViewHolder, position: Int) {
         val form: FormData = formList[position]
         form.also {
-            holder.formCategoryTextView.text = it.formCategory.capitalize()
+            holder.formCategoryTextView.text = it.formCategory.capitalize(Locale.ROOT)
```

Comment:
Can we use `Locale getDefault()` instead to get the current locale set on the device.